games/ludo/master.py

```diff
@@ -519,18 +585,25 @@ def create_game_master(
             player_models (list[Model]): contains two player models, being of
                                          different child classes depending on
                                          the game variant
+            chain_of_thought (bool): allows for chain-of-thought functionality
+                                     if True
+            reprompting (bool): allows for reprompting of the LLM if True
 
         Returns:
             LudoGameMaster: instantiated LudoGameMaster object
         """
-        return LudoGameMaster(experiment, player_models)
+        return LudoGameMaster(
+            experiment,
+            player_models,
+            self.chain_of_thought,
```

Comment:
Do we need CoT and reprompting as a object attribute? Or can we pass it along with the game instance json within the experiment?